### DISCUSSION TITLE
mail__from environment variable missing/not working

### DISCUSSION THREAD

**Discussion by Askotion** [docker-library/ghost#347] [General]
Hi,

i can't figure out how to get the mail__from env variable to work. It seems like its not available anymore?
Im using my own smtp server for this.

```yaml
    environment:
      mail__from: "Test <test@example.com>"
```

I still get this error which clearly stats its using the default from address.

Failed to send email. Reason: Can't send mail - all recipients were rejected: 553 5.7.1 <noreply@blog.example.com>: Sender address rejected: not owned by user test@example.com.

I tried setting it with "ghost config mail__from "test <test@example.com>"". But no luck.
Is this feature even available anymore?

Best

**Comment by pascalandy** [comment]
search in old issues for "config.production.json"

**Comment by Askotion** [comment]
> search in old issues for "config.production.json"

Yes i did that. But as far as i understand the config.production.json is not being touched when using environment variables through docker.  Mine is just the default one without my configured variables. 

The other variables such as email user and password are working properly but just the mail from doesn't. Imo thats a bug.

Best

**Comment by pascalandy** [comment]
If you feel it's a bug, you should check the ghost forums. Also, have you tried to run Ghost locally using the ghost cli ?

**Comment by Tauromachian** [comment]
I am having this same problem. Ghost will send the same noreply@mydomain.com and I can't edit it with the mail__from variable to something like support@mydomain.com for example

**Reply by tianon** [reply]
According to https://docs.ghost.org/config#email-addresses, `mail__from` (with two underscores) _should_ be correct :thinking:
